src/server.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SERVER_H
+#define SERVER_H
+
+#include <boost/asio.hpp>
+#include <array>
+
+class Server
+{
+    public:
+        Server(boost::asio::io_service& io_service);
+
+    private:
+        void start_receive();
+        void handle_receive(const boost::system::error_code& error, std::size_t /*bytes_transferred*/);
```

Comment:
It is not necessary to put parameter names in comments here. These are declarations.

Reply:
Same below.